hs_core/views/utils.py

```diff
@@ -43,12 +43,12 @@ def authorize(request, res_id, edit=False, view=False, full=False, superuser=Fal
     except ObjectDoesNotExist:
         raise NotFound(detail="No resource was found for resource id:%s" % res_id)
 
-    has_edit = res.edit_users.filter(pk=user.pk).exists()
-    has_view = res.view_users.filter(pk=user.pk).exists()
-    has_full = res.owners.filter(pk=user.pk).exists()
+    has_edit = res.raccess.edit_users.filter(pk=user.pk).exists()
+    has_view = res.raccess.view_users.filter(pk=user.pk).exists()
+    has_full = res.raccess.owners.filter(pk=user.pk).exists()
 
     authorized = (edit and has_edit) or \
-                 (view and (has_view or res.public)) or \
+                 (view and (has_view or res.raccess.public)) or \
                  (full and has_full) or \
                  (superuser and user.is_superuser)
```

Comment:
The admin flag distinguishes someone who has permission to update HydroShare permissions from someone who has general Django superuser. It is possible -- with this mechanism -- to have a non-superuser who can administer and make changes to the access control subsystem only. 

I realize that the code base currently has no such concept, and utilizes superuser permissions to do this. However, the current access control system requires someone to have admin -- and not just superuser -- to change access control, e.g., in share_*_with_*. 

So, one can either: 
a) set UserAccess.admin=True for Django superusers, and all will be well, or 
b) recode access control so that User.is_superuser is utilized instead of UserAccess.admin. 

Myself, I thought it handy to have a privilege level less than superuser that I could, e.g., hand out to graduate students curating the site. No strong feelings either way.